var.tfvars

```diff
@@ -16,9 +16,11 @@ bastion = {
   count         = 1
 }
 tang = {
-  image_id      = "27ebd00f-cbec-4e27-993d-56e4bd441584"
-  instance_type = "base-ocp-squad-tiny"
-  count         = 3
+  image_id          = "27ebd00f-cbec-4e27-993d-56e4bd441584"
+  instance_type     = "base-ocp-squad-tiny"
+  count             = 3
+  data_volume_count = 1
+  data_volume_size  = 10
```

Comment:
Let's be sure the docs are updated with the variable description. 
Also variables.tf if that is updated, let's be sure to describe the volume size and count

Reply:
Let's lock in the data_volume_count to 1

Reply:
Added validation for data_volume_count and it will throw error when value provided is other than 1.

Reply:
data_volume_count ... where is this validation? I'd expect it to be in variables.tf like this... https://github.com/IBM/powervm-tang-server-automation/blob/main/variables.tf#L96